tests/storage/memorydump.go

```diff
@@ -307,9 +307,14 @@ var _ = SIGDescribe("Memory dump", func() {
 
 	memoryDumpVirtctlCreatePVC := func(name, namespace, claimName string) {
 		By("Invoking virtctl memory dump with create flag")
+		storageClass, exists := libstorage.GetRWOFileSystemStorageClass()
```

Comment:
otherwise, this results in virtctl creating a memory dump PVC with the default storage class
https://github.com/kubevirt/kubevirt/blob/aeb6ef0526244b143fa6cccf349a14199828c916/tests/storage/memorydump.go#L317